.github/workflows/pr-checks.yaml

```diff
@@ -68,7 +68,7 @@ jobs:
     runs-on: ubuntu-20.04
 
     env:
-      EXTRA_BEHAVE_ARGS: "--tags=~@knative --tags=~@openshift --tags=~@examples"
+      EXTRA_BEHAVE_ARGS: "--tags=~@knative --tags=~@openshift"
```

Comment:
I would revert this line, so that when we examples, we do not need to touch this line again.

Reply:
Good point, thx - reverted